extension/src/sidebar.js

```diff
@@ -427,10 +455,23 @@ async function* _bindSidebarData(response: Visits) {
 
         // TODO show total counts?
         // TODO if too many tags, just overlap on the seconds line
-        const tag_c = binder.makeChild(all_tags_c, 'span', ['src', asClass(tag)])
-        binder.makeTchild(tag_c, `${asClass(tag)} (${count})`);
+        const tag_c = binder.makeChild(filtersEl, 'span', ['src', 'button', asClass(tag)])
+		const tag_c_name = binder.makeChild(tag_c, 'span', ['src-name'])
+		const tag_c_count = binder.makeChild(tag_c, 'span', ['src-count'])
+        binder.makeTchild(tag_c_name, `${asClass(tag)}`);
+		binder.makeTchild(tag_c_count, `${count}`);
         // TODO checkbox??
         tag_c.addEventListener('click', () => {
+			// set active status to filter buttons
+			// 1) reset status from all sources
+			filtersEl.childNodes.forEach(filter_element => {
+				if (filter_element.nodeType == Node.ELEMENT_NODE) {
+					filter_element = ((filter_element: any): HTMLElement);
+					filter_element.classList.remove('active');
+				}
+			});
+			// 2) set active to clicked source
+			tag_c.classList.add('active');
```

Comment:
code seems misaligned in this block, can you shift it one space to the left?

Reply:
done